src/test/CMakeLists.txt

```diff
@@ -140,6 +145,10 @@ add_executable(ceph_kvstorebench
   ${kvstorebench_srcs}
   )
 target_link_libraries(ceph_kvstorebench librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS})
+install(TARGETS
```

Comment:
please don't wrap line like this. if we have multiple targets, we might want to have 1 item for each line.